demo/cloudify_server

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+REPL = true
+
+require_relative "../loader"
+
+def get_input(msg, default = nil)
+  prompt = "#{msg}#{default.nil? ? "" : " [default: #{default}]"}: "
+  input = ""
+
+  while input.empty?
+    print prompt
+    input = gets.chomp
+    input = default if input.empty? && !default.nil?
+  end
+
+  input
+end
+
+hostname = get_input("Enter hostname or IP address")
+location = get_input("Enter location", "hetzner-hel1")
+
+strand = Prog::Vm::HostNexus.assemble(hostname, location: location)
+
+puts "Waiting public SSH keys\n\n"
+until (ssh_key = strand.reload.sshable.keys.map(&:public_key).first)
+  sleep 2
+end
+puts "Add following public SSH key to '/root/.ssh/authorized_keys' on your machine\n\n"
+puts ssh_key
+
+print "\n\nPress enter when you added them to your server\n\n"
```

Comment:
Hey Enes,

This pull request generally looks good to me. I have the following code review comments.

1. Small edits to messages that we display to the user

"Press enter when you added them to your server" -> "Press enter when you added the above SSK key on your server"
"Waiting server to be cloudified" -> "Waiting for server to be cloudified"

2. In the script, we say location. In practice, the user is entering provider and location. Should we make that change? (minor)

Similarly, in the UI, should we default to Helsinki? Or at least make sure we either use country or city names in our UI and demo?

3. If I enter the wrong host name or the wrong SSH key, the script looks forever. Should we add a timeout?
4. This isn't related to the PR, but just my curiosity. What does REPL mean and what does it do?

Once you resolve these comments as you see fit, please feel free to merge the changes.

Reply:
1. I updated messages
2. I changed location input to select instead of freeform text. If user doesn't know supported providers and locations freeform text isn't very useful. Select list looks more user-friendly to me. I used similar options to similar to our old VmHost UI.
```
1. Hetzner Finland
2. Hetzner Germany
3. DataPacket Istanbul

Select provider and location [default: 1]: 2
```
3. Added 10 minutes time out with following message. I put GitHub discussion page to increase community interaction.
```
Could not cloudify server in 10 minutes. Probably something went wrong.
Last state: wait_bootstrap_rhizome. Server ID for debug: b61f38d3-8bb1-8331-a264-f3ed899e3d4c
Please check your hostname/IP address and be sure that you added the correct public SSH key to your server.
You can ask for help on GitHub disscussion page: https://github.com/ubicloud/ubicloud/discussions
```
4. REPL stands for simple interactive computer programming environment. We set it true for pry and other user scripts. If REPL is true, we use different logging preferences to avoid excessive log printing or make debugging easier etc.

Reply:
Since I changed lots things, It might be good to get an another review for provider&location select and timeout message. @ozgune